stdlib/sqlite3/dbapi2.pyi

```diff
@@ -453,6 +453,8 @@ if sys.version_info >= (3, 11):
         def seek(self, __offset: int, __origin: int = ...) -> None: ...
         def __len__(self) -> int: ...
         def __enter__(self: Self) -> Self: ...
-        def __exit__(self, __typ: object, __val: object, __tb: object) -> Literal[False]: ...
+        def __exit__(
+            self, __typ: type[BaseException] | None, __val: BaseException | None, __tb: TracebackType | None
+        ) -> Literal[False]: ...
```

Comment:
This signature was previously discussed in https://github.com/python/typeshed/pull/7625#discussion_r851630434: the parameters here really do take any object